src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,16 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given thread. The following restrictions apply:
+
+- Active and inactive chats can be tagged.
+- One or more tags can be added to the specific **thread**.
```

Comment:
this method allows adding a single tag in a single call so maybe it's better to emphasize this as a thread property? something like "a thread can have multiple tags"?

Reply:
one more thing, we should update "chat" data structure and add "tags" array to the thread there

Reply:
If I understand correctly - What do you think to write it directly - One tag can be added in one call, but one thread can have multiple tags'? 🤔 

"chat" data structure and add "tags" array to the thread there - I will create a task for that in Jira and investigate 😄

Reply:
no need for a separate task 😉 just add it [here](https://github.com/livechat/livechat-public-docs/blob/master/payloads/messaging/v3.4/agent-chat-api/other-structures/chat.json) cc @ale-kacperczyk 

>One tag can be added in one call, but one thread can have multiple tags

idk, the interface kinda forces you to provide a single tag, and I don't like stating obvious things :P

Reply:
1. It's a dilemma between "Write it explicitly in the description so that people know right away" and "Give people a minute to analyze request parameters so that they figure out what can be sent themselves".

> One tag can be added in one call, but one thread can have multiple tags

"One tag can be added in one call" can be concluded from the request parameters, so we would be restating what's already expressed in the table. That leaves us with the information that "a thread can have multiple tags", which is true, but would be more fitting for a general overview of tags (like the PR @gluszczyk prepared for v3.5). It's only loosely related to this method.

Taking all into account, I think that maybe it's best to remove this line altogether?

2. We will update the chat data structure in a separate PR right after we're done with this one. That's just a more convenient way to do it for us.

Reply:
Yes, I agree that if someone analyzes the request parameters then the fact that 'one tag can be added in one call' can be figured out. 
I would say that the info that 'a thread can have multiple tags' is important, but you may be right that it fits more the general overview of tag.
I will remove that info completely:)